shared/reducers/fs.js

```diff
@@ -64,9 +65,9 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
 
         return [
           path,
-          // Since both `folderListLoaded` and `favoritesLoaded` can change
-          // `pathItems`, we need to make sure that neither one clobbers the
-          // other's work.
+          // Since `folderListLoaded`, `favoritesLoaded`, and `loadResetsResult`
+          // can change `pathItems`, we need to make sure that neither one
+          // clobbers the others' work.
```

Comment:
we have a helper in chat that does this and makes it more clear, might be worth doing here, something like updatePathItem(existing, newItem) or something

Reply:
I think I'm going to leave it for now, until we need it one more time.